src/fiatExchanges/CoinbasePaymentSection.test.tsx

```diff
@@ -36,17 +43,67 @@ describe('CoinbasePaymentSection', () => {
       )!,
       appId: FAKE_APP_ID,
       analyticsData: mockProviderSelectionAnalyticsData,
+      tokenId: mockCeloTokenId,
     }
-    mockStore = createMockStore()
+    mockStore = createMockStore({
+      tokens: {
+        tokenBalances: mockTokenBalances,
+      },
+    })
   })
 
+  it.each([
+    {
+      tokenId: mockCeloTokenId,
+      supportedNetworks: ['celo'],
+      assets: ['CGLD'],
+    },
+    {
+      tokenId: mockEthTokenId,
+      supportedNetworks: ['ethereum'],
+      assets: ['ETH'],
+    },
+  ])(
+    'calls generateOnRampURL with correct params',
```

Comment:
nit: maybe adding a `testName` to the above array and using that so that we see which network is used from the test output.

Reply:
+1, could also just add the token Id. E.g., 
```suggestion
    'calls generateOnRampURL with correct params for $tokenId',
```